### ISSUE TITLE
Rename recommendation

### ISSUE THREAD

**Issue by neruthes** [DimensionDev/linkedin_driver#13]
To `Metadrive-Driver-LinkedIn`

cc @yisiliu @mindey

**Comment by mindey** [comment]
@neruthes makes sense, in context of [the convenitions](https://github.com/DimensionDev/Meta/blob/master/articles/1--github-repo-naming.md).

In addition, since we will have a lot of various drivers, I have even dedicated a separate github user fork and hold them -- [github.com/drivernet](https://github.com/drivernet).
